sympy/vector/coordsysrect.py

```diff
@@ -14,7 +15,7 @@ class CoordSysCartesian(Basic):
     """
```

Comment:
I will do it at the end or even in another PR. It introduces many changes in docstring so it mess up the code.